test-packages/nightly-tests/generator/openapi-negative-case.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { resolve } from 'path';
+import execa from 'execa';
+import {
+  testDir,
+  testOutputRootDir,
+  testResourcesDir
+} from '../../../test-resources/generator';
+
+describe('openapi negative tests', () => {
+  // TODO Put test back in once the spec validation is put back in.
+  xit('should fail on generation for faulty spec file', async () => {
+    const output = resolve(
+      testOutputRootDir,
+      'openapi-negative',
+      'faulty-specification'
+    );
+    await expect(
+      execa(
+        'npx',
+        [
+          'generate-openapi-client',
+          '-i',
+          resolve(testResourcesDir, 'test-resources', 'faulty-openapi'),
+          '-o',
+          output,
+          '--clearOutputDir'
+        ],
+        { cwd: __dirname }
+      )
+    ).rejects.toThrowError('set failed method');
+  }, 120000);
+
+  it('should fail on transpilation on faulty tsconfig', async () => {
+    const output = resolve(
+      testOutputRootDir,
+      'openapi-negative',
+      'transpilation-failed'
+    );
+    await expect(
+      execa(
+        'npx',
+        [
+          'generate-openapi-client',
+          '-i',
+          resolve(testDir, '../openapi-service-specs'),
+          '-o',
+          output,
+          '--tsConfig',
+          resolve(testResourcesDir, 'faulty-openapi-tsconfig', 'tsconfig.json')
+        ],
+        { cwd: __dirname }
+      )
+    ).rejects.toThrowError("Cannot find name 'URL'");
```

Comment:
Could you please explain what's wrong with the tsconfig and why the error message is this?

Reply:
I removed the dom in the libs which leads to the URL class not beeing there.